Makefile

```diff
@@ -1,25 +1,183 @@
-NAME:=vrt
-LICENSE:="OSI Approved :: Apache Software License v2.0"
-AUTHOR:="Afiniate, Inc."
-ORGANIZATION:="afiniate"
-HOMEPAGE:="https://github.com/afiniate/vrt"
+# =============================================================================
+# Build Management Knobs
+# =============================================================================
 
-DEV_REPO:="git@github.com:afiniate/vrt.git"
-BUG_REPORTS:="https://github.com/afiniate/vrt/issues"
+PARALLEL_JOBS ?= 2
```

Comment:
maybe. I would say w can do a quick search and if we find something easy great but not to put a hard requirement on it.